tests/test-cmd/test.build.js

```diff
@@ -37,6 +37,31 @@ describe('build', () => {
     );
   });
 
+  it('zips a package with custom file extension', () => {
+    let zipFile = new ZipFile();
+
+    return withTempDir(
+      (tmpDir) =>
+        build({
+          sourceDir: fixturePath('minimal-web-ext'),
+          artifactsDir: tmpDir.path(),
+          extension: 'blah',
+        })
+        .then((buildResult) => {
+          assert.match(buildResult.extensionPath,
+                       /minimal_extension-1\.0\.blah$/);
```

Comment:
You could end the test here. I don't think you need to make assertions about the contents of the package since that's covered by other tests.